code/__DEFINES/atmospherics.dm

```diff
@@ -161,10 +161,14 @@
 #define ATMOS_TANK_O2				"o2=100000;TEMP=293.15"
 #define ATMOS_TANK_N2				"n2=100000;TEMP=293.15"
 #define ATMOS_TANK_AIRMIX			"o2=2644;n2=10580;TEMP=293.15"
+
 //LAVALAND
 #define LAVALAND_EQUIPMENT_EFFECT_PRESSURE 50 //what pressure you have to be under to increase the effect of equipment meant for lavaland
 #define LAVALAND_DEFAULT_ATMOS "o2=14;n2=23;TEMP=300"
 
+//SNOSTATION
+#define ICEMOON_DEFAULT_ATMOS "o2=22;n2=82;TEMP=180"
```

Comment:
Looking for suggestions on this gas mix, currently this is enough to make you chilly (light blue thermometer) with a winter coat, and freezing (taking slow cold damage) without a coat. Internals are currently required.